README.md

```diff
@@ -9,38 +9,63 @@ A practical functional library for Javascript programmers.
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/ramda/ramda?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 
-Goals
------
+Why Ramda?
+----------
 
 <img src="http://ramda.jcphillipps.com/logo/ramdaFilled_200x235.png" 
-     width="200" height="235" align="left" hspace="12" />
-Using this library should feel as much like using Javascript as 
-possible.  Of course it's functional Javascript, but we're not 
-introducing lambda expressions in strings, we're not borrowing consed 
+     width="170" height="190" align="right" hspace="12" />
+
+Ramda makes it simple for you to build complex logic through
+functional composition. This means sticking simple functions together.
+It is designed to help you get you job more elegantly with simpler code.
+Ramda does not mutate your state. Any data you throw at Ramda is safe from
+hidden state changes, and this is a very powerful weapon to battle complexity.
+
+What's Different?
+-----------------
+
+There are already excellent libraries with functional flavor,
+such as _Underscore_ and _Lodash_. Ramda includes all of the favorite
+list-manipulation functions you expect, e.g. map, filter, reduce, find, and so forth.
+However, Ramda is significantly different from libraries like _Underscore_ and _Lodash_.
+
+The primary distinguishing features of Ramda are:
+
+1. Ramda emphasizes a purer functional style where practical. Immutability and side-effect free functions
+are at the heart of it's design philospohy. This can help you get the job done with simple, elegant code.
+2. Ramda takes the function first, and the data last for effective currying. In a nutshell, the combination of currying and function-first enables the developer to compose functions with very little code (often in a “point-free” fashion), before finally passing in the data.
+3. Ramda methods are automatically curried. This auto-currying makes it easy to compose functions to create new functions. Because the API is function-first, data-last, you can continue composing and composing until you build up the function you need before dropping in the data.
```

Comment:
How about

> 2. Ramda functions are automatically curried.  This allows you to easily build up new functions from old ones simply by not supplying the final parameters.
 > 3. The parameters to Ramda functions are arranged to make it convenient for currying.  The data to be operated on is generally supplied last.
>
> The last two points together make it very easy to build functions as sequences of simpler functions, each of which transforms the data and passes it along to the next.  That is the style coding Ramda is designed to support.